maintenance/tables.sql

```diff
@@ -1098,10 +1098,7 @@ CREATE TABLE /*_*/interwiki (
   -- insertion.
   iw_url blob NOT NULL,
 
-  -- The URL of the file api.php
-  iw_api blob NOT NULL,
-
-  -- The name of the database (for a connection to be established with wfGetLB( 'wikiid' ))
```

Comment:
In code it's used only via api
My script already checked 2/3 wikis.
None of them use those columns.